### ISSUE TITLE
Questions about DeepL auto translation and Whisper

### ISSUE THREAD

**Issue by teijiIshida** [SubtitleEdit/subtitleedit#8457]
I have several questions if you don't mind.

In this Settings picture, the URL leads to 404 Not Found. 
![settings](https://github.com/SubtitleEdit/subtitleedit/assets/33733459/88ec343f-69e4-4217-8551-e2f0b20b91d2)

Should I change it to this page?
![deepl api](https://github.com/SubtitleEdit/subtitleedit/assets/33733459/7ac7d81a-5c2b-4c6a-bf69-cfe7aac10d68)

In the "Audio to text (Whisper)", there is a Translate to English checkbox. If I put my DeepL API key in the "Auto translate" settings in the first picture, will it use that to translate to English?
![whisper](https://github.com/SubtitleEdit/subtitleedit/assets/33733459/5a6e9c27-c16e-495b-af59-79954e313fd8)

**Comment by niksedk** [comment]
No, keep the DeepL url as is (it's a base url for API endpoint - see more here: https://developers.deepl.com/docs/api-reference/translate )

Whisper translate does it's own translating, but it's probably better to use DeepL.
